ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -130,20 +129,6 @@ StatusDialog {
                             networkFilter.setChain(root.swapInputParamsForm.selectedNetworkChainId)
                     }
                 }
-
-                Connections {
-                    target: root.swapInputParamsForm
-                    function onSelectedNetworkChainIdChanged() {
-                        networkFilter.setChain(root.swapInputParamsForm.selectedNetworkChainId)
-                    }
-                }
-
-                Connections {
-                    target: root.swapInputParamsForm
-                    function onSelectedNetworkChainIdChanged() {
-                        networkFilter.setChain(root.swapInputParamsForm.selectedNetworkChainId)
-                    }
```

Comment:
Duplicate code; bad merge? But moved above to the toplevel `Connections` so that we first call `setChain` and then re-eval the selected token (to get the correct max balance etc.)